### PR TITLE
JAMES-2446 Enhance Cassandra tests

### PR DESCRIPTION
**Pull request by aduprat**
Reviews are not expected on this pull request.

### CODE REVIEW

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by chibenwa** [comment]
Force pushed to solve conflict

**Comment by chibenwa** [comment]
Was green

File: backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraWaitStrategy.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public CassandraWaitStrategy(GenericContainer<?> cassandraContainer, Duration ti
 
     @Override
     public void waitUntilReady(WaitStrategyTarget waitStrategyTarget) {
-        Unreliables.retryUntilTrue(Ints.checkedCast(timeout.getSeconds()), TimeUnit.SECONDS, () -> {
+        Unreliables.retryUntilTrue(Ints.checkedCast(timeout.toMillis()), TimeUnit.MILLISECONDS, () -> {
```
[the remark this answers is not in the document]
Comment (chibenwa) [reply backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraWaitStrategy.java]: Agree, let skip this commit.

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**